File: four_max/four_max.c
#include "stdio.h"

int a, b, c, d;

int main() {
    scanf("%d%d%d%d", &a, &b, &c, &d);
    if (a > b && a > c && a > d)
        printf("max val: %d\n", a);
    else if (b > c && b > d)
        printf("max val: %d\n", b);
    else if (c > d)
        printf("max val: %d\n", c);
    else
        printf("max val: %d\n", d);

    return 0;
}

File: four_max/four_max.py
# author : syqwq

if __name__ == "__main__":
    a, b, c, d = map(eval, input().split())
    if a>b and a>c and a>d:
        print("max val: {}".format(a))
    elif b>c and b>d:
        print("max val: {}".format(b))
    elif c>d:
        print("max val: {}".format(c))
    else:
        print("max val: {}".format(d))
